keps/sig-auth/3130-kms-observability/README.md

```diff
@@ -123,6 +123,7 @@ Integration test covering:
 
 1. Logging of UID in kube-apiserver
 2. UID in the `EncryptRequest` and `DecryptRequest`
+3. UID set to nil in the `EncryptRequest` and `DecryptRequest` when the feature gate is disabled
```

Comment:
Also: confirm that this results in byte equivalent data on the wire when compared to a 1.23 API server.